artifacts/openemr/openemr-deployment.yaml

```diff
@@ -59,7 +59,7 @@ spec:
           value: "redis"
         - name: SWARM_MODE
           value: "yes"
-        image: "openemr/openemr:5.0.2"
+        image: "quay.io/openemr/openemr"
```

Comment:
i suggest we specify an image tag

Reply:
Would `latest` work?

Reply:
that is not recommended. imagine we build a new image, all instances are updated and we find a bug.

Reply:
i think we should build images per git commit. if the current pipelines are not configured that way, my team can help with that!

Reply:
i don't want to block on this, so i'm +1 to merge to unblock further work.
but i do suggest we think this over. cc @jfchevrette who proposed to work on this earlier today.

Reply:
I received an "Image quay.io/openemr/openemr runs as the root user which might not be permitted by your cluster administrator." when deploying to my OKD 3.11 cluster.

Is it possible to run as a non-root user?

Reply:
All these are good comments, better tagging, not run as root, and more official image location. I'm going to go ahead and merge it though because switching images is a blocker for a lot of other things like figuring out the autoconfig bit.